### ISSUE TITLE
Is there any chance to implement support for messages (aka toots)?

### ISSUE THREAD

**Issue by paulcable** [Automattic/wordpress-activitypub#53]
Direct/Private/Unlisted/Public

1) If somebody sends you a message (mentioning on the fediverse) to redirect it to email.
2) To add a simple form (in Wordpress menu or dashboard) where you can choose message type (private, public, etc), write an answer (without posting it on Wordpress).

**Comment by Cambridgeport90** [comment]
I don't think this is meant to be an actual messaging client within Wordpress, is it?

**Comment by lapineige** [comment]
> Direct/Private/Unlisted/Public
> 
>     1. If somebody sends you a message (mentioning on the fediverse) to redirect it to email.
> 
>     2. To add a simple form (in WordPress menu or dashboard) where you can choose message type (private, public, etc), write an answer (without posting it on WordPress).

I would be greatly in favor of 1), as people could mention us (and maybe with "private" information) and newer have a reply because we never were even aware of being mentioned.
I don't know if redirecting to email is the best solution, but maybe it's the one that require the less code & efforts.

Regarding 2), I don't think it is the purpose of such a WordPress plugin to implement full messaging capability…

**Comment by lapineige** [comment]
Could we keep this open ?

**Comment by pfefferle** [comment]
keep

**Comment by HammyHavoc** [comment]
I'm a big fan of GitHub email notifications with the ability to reply to an Issue or PR via email. Would love the same for this. I don't necessarily even want another app, I just want it to sit on top of where I do most of my communication already, which is email for asynchronous, and Matrix for real-time.

**Comment by HammyHavoc** [comment]
Commenting to keep it open.

**Comment by HammyHavoc** [comment]
Commenting to keep it open.

**Comment by pfefferle** [comment]
Keep!

**Comment by thomasjwebb** [comment]
Shouldn't this be implemented in the friends plugin? I would really like to be able to send and receive messages because I need it to interact with bridgy fed but it has other uses too of course and it can be confusing if someone sends your blog a message and you can't even see it.

**Comment by pfefferle** [comment]
We did a first step in this direction by forwarding DMs via E-Mail. But they are definitely on our list!

**Comment by thomasjwebb** [comment]
Oh yeah correction: I did get an e-mail of the message I sent. So it's just going the other way I can't do right now.

**Comment by HammyHavoc** [comment]
Still relevant.

**Comment by Jiwoon-Kim** [comment]
https://github.com/samunohito/misskey-github-notifier-next

**Comment by HammyHavoc** [comment]
Still interested.
